src/components/notifications/AppSelector/index.tsx

```diff
@@ -124,47 +130,59 @@ const AppSelector: React.FC = () => {
           )}
         </div>
         <div className="AppSelector__wrapper">
-          <Label color="main">Subscribed</Label>
+          {!loading && filteredApps?.length > 0 ? <Label color="main">Subscribed</Label> : null}
           <ul className="AppSelector__list">
             {loading
               ? Array(3)
                   .fill(<LinkItemSkeleton />)
                   .map(x => x)
               : null}
             {!loading &&
-              filteredApps?.map(app => (
-                <AnimatePresence>
-                  <m.div initial={{ opacity: 0 }} exit={{ opacity: 0 }} animate={{ opacity: 1 }}>
-                    <NavLink
-                      key={app.topic}
-                      to={`/notifications/${app.topic}`}
-                      className="AppSelector__link-item"
-                    >
-                      <div className="AppSelector__notifications">
-                        <div className="AppSelector__notifications-link">
-                          <div className="AppSelector__link-logo">
-                            <img
-                              src={app.metadata.icons?.[0] || '/fallback.svg'}
-                              alt={`${app.metadata.name} logo`}
-                              onError={handleImageFallback}
-                              loading="lazy"
-                            />
-                          </div>
-
-                          <div className="AppSelector__link__wrapper">
-                            <Text className="AppSelector__link__title" variant="small-500">
-                              {app.metadata.name}
-                            </Text>
-                            <Text className="AppSelector__link__subtitle" variant="small-500">
-                              {app.metadata.description}
-                            </Text>
+              filteredApps?.map(app => {
+                const unverified = isUnVerified(app, projects)
+
+                return (
+                  <AnimatePresence>
+                    <m.div initial={{ opacity: 0 }} exit={{ opacity: 0 }} animate={{ opacity: 1 }}>
+                      <NavLink
+                        key={app.topic}
+                        to={NAVIGATION.notifications.topic(app.topic)}
```

Comment:
It'd be nice to use a more explicit name
```suggestion
                        to={NAVIGATION.notifications.getPathByTopic(app.topic)}
```

Reply:
The goal of naming here is to have the same function name as the pathname to match fn names with variable names:

```js
/notifications -> NAVIGATION.notifications
/notifications/{topic} -> NAVIGATION.notifications.topic("TT")
```

When we have inner pages under `/notifications/{topic/*` this might make more sense in terms of readability:

```js
/notifications/{topic}/settings -> NAVIGATION.notifications.topic("TT").settings
/notifications/{topic}/settings/billing -> NAVIGATION.notifications.topic("TT").settings.billing
```

`getPathByTopic ` still make sense tho

Reply:
Thanks for the context, it's a bit unclear what the `topic` function does just by its name, that's why I suggested to rename it.